Example doesn't work using Rollup CLI

### Expected behavior

When I follow the instructions in the example for building via Rollup CLI, I expect it to output a bundle to the `dist` directory.

### Actual behavior

It throws an error:

```[!] Error: Cannot find module '../dist/rollup-plugin-vue.common.js'```

So, the first thing that's missing is -- the instructions should say you have to `cd .. && npm install && npm run build` first before running the example.

Then, I got another error:

```
[!] Error: You must specify options.format, which can be one of 'amd', 'cjs', 'es', 'iife' or 'umd'
```

There's also a warning:

```
(!) Some options have been renamed
https://gist.github.com/Rich-Harris/d472c50732dab03efeb37472b08a3f32
--output is now --output.file
```

So it seems the API has changed since this example was written. Just wanted to give a heads up :)

### Steps to reproduce the behavior

Follow the instructions in `example/README.md`.

--- a/example/README.md
+++ b/example/README.md
@@ -18,5 +18,5 @@ ls dist
 ```
 cd example
 npm i -g rollup
-rollup -c --input Hello.vue --output dist/bundle.js
+rollup -c --input Hello.vue
 ```

--- a/example/rollup.config.js
+++ b/example/rollup.config.js
@@ -2,6 +2,18 @@
 
 const vue = require('../dist/rollup-plugin-vue.common.js');
 const buble = require('rollup-plugin-buble');
+
 export default {
-    plugins: [ vue({compileTemplate: true}), buble() ],
+  name: 'helloRollupVue',
+  output: {
+    file: 'dist/bundle.js',
+    format: 'umd'
+  },
+  plugins: [
+    vue({
+      compileTemplate: true,
+      css: 'dist/bundle.css'
+    }),
+    buble()
+  ]
 };

